src/frontend/typer/behandlingstema.ts

```diff
@@ -40,29 +38,17 @@ export interface IBehandlingstema {
 }
 
 export const behandlingstemaer: Record<Behandlingstema, IBehandlingstema> = {
-    NASJONAL_ORDINÆR: {
-        kategori: BehandlingKategori.NASJONAL,
-        underkategori: BehandlingUnderkategori.ORDINÆR,
-        navn: 'Nasjonal ordinær',
-        id: 'NASJONAL_ORDINÆR',
-    },
-    NASJONAL_UTVIDET: {
+    NASJONAL: {
         kategori: BehandlingKategori.NASJONAL,
-        underkategori: BehandlingUnderkategori.UTVIDET,
-        navn: 'Nasjonal utvidet',
-        id: 'NASJONAL_UTVIDET',
-    },
-    EØS_ORDINÆR: {
-        kategori: BehandlingKategori.EØS,
-        underkategori: BehandlingUnderkategori.ORDINÆR,
-        navn: 'EØS ordinær',
-        id: 'EØS_ORDINÆR',
+        underkategori: BehandlingUnderkategori.ORDINÆR, //todo: avklaring, skal vi ha med underkategori her?
```

Comment:
Skal vi ha med underkategori? eller kom vi frem til noe her i tidligere diskusjoner?

Reply:
Hmm, det husker jeg ikke 🤔 Kanskje Eivind har et svar?

Reply:
Vi trenger det ikke for behandlingen, men det kan hende vi må tolke BehandlingKategori, om det er nasjonalt eller eøs, fra BehandlingUnderkategori-feltet i oppgavene vi får fra Gosys. Tenker vi kan fjerne det her og håndtere oppgavene fra gosys for seg selv.

Reply:
Fjerner denne i en egen PR, ser at det blir en endringer 😊